notebooks/official/prediction/sdk2_remote_prediction.ipynb

```diff
@@ -0,0 +1,1421 @@
+{
```

Comment:
<p>Wonder why we don't need to register the model for PyTorch Lightning like other frameworks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
Registering a PyTorch Lightning model appears not yet supported. I removed the corresponding code.